azkaban-common/src/main/java/azkaban/constants/ServerInternals.java

```diff
@@ -26,5 +26,12 @@
   public static final String AZKABAN_EXECUTOR_PORT_FILENAME = "executor.port";
 
   public static final String AZKABAN_SERVLET_CONTEXT_KEY = "azkaban_app";
-  
+
+
+  // Memory check retry interval when OOM in ms
+  public static final long MEMORY_CHECK_INTERVAL = 1000*60*10;
```

Comment:
Discussed with @HappyRay  offline. We agreed that let's first start with short interval to avoid job long starving. We can adjust the parameter if we observe any outstanding issue.